CodeEdit/Features/Settings/Pages/TextEditingSettings/TextEditingSettingsView.swift

```diff
@@ -90,4 +91,14 @@ private extension TextEditingSettingsView {
                 .foregroundColor(.secondary)
         }
     }
+
+    private var letterSpacing: some View {
+        Stepper(
+            "Letter Spacing",
+            value: $textEditing.letterSpacing,
+            in: 1.0...2.0,
```

Comment:
It would be great if this range was a little bigger. I think from 0.5 to 2.0 would be good.

Reply:
Personally I'm not too fond of the way `0.5` letter spacing looks maybe an extension in the other way (`1.0` to `3.0`) would be better, thoughts @thecoolwinter?
<img width="264" alt="shot 1" src="https://github.com/CodeEditApp/CodeEdit/assets/128280019/0ca516b7-fc80-4ddc-8f7f-4dd5673d6b2e">

Reply:
I think that is fine just in case there is a wider spaced monospaced font. If you really insist, we might go with 1.75 - 2.

Reply:
Actually 0.5 to 2 seems fine with a different font, i'll commit the changes now.